src/target/llvm/codegen_hexagon.cc

```diff
@@ -636,7 +636,10 @@ bool UsesExportABI(const PrimFunc& f) {
   return false;
 }
 
-__attribute__((unused)) std::ostream& operator<<(std::ostream& os, const llvm::Module& m) {
+#ifdef __GNUC__
+__attribute__((unused))
+#endif
+std::ostream& operator<<(std::ostream& os, const llvm::Module& m) {
```

Comment:
I think we could change it into `DMLC_ATTRIBUTE_UNUSED`

Reply:
I agree